src/core/ctf.cpp

```diff
@@ -105,7 +105,7 @@ void CTF::Init(float wanted_acceleration_voltage_in_kV, // keV
                float wanted_particle_shift_y_in_angstroms, // A
                float wanted_sample_thickness_in_nm) // nm
 {
-    wavelength                    = WavelengthGivenAccelerationVoltage(wanted_acceleration_voltage_in_kV) / pixel_size_in_angstroms;
+    wavelength                    = ReturnWavelenthInAngstroms(wanted_acceleration_voltage_in_kV) / pixel_size_in_angstroms;
```

Comment:
Just a name change to more clearly indicate the return value and its unit?

Reply:
It is also a slightly different function and defined in non_wx_functions.h rather than ctf.cpp as the Wavelength is used in other compilation units.